src/main/java/ChatReques5.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
```

Comment:
Redundant annotations. There are multiple Lombok annotations used (@Setter, @Getter, @ToString, @Data) which provide overlapping functionality. It is recommended to use only one that suits the requirements.